packages/flutter/test/animation/iteration_patterns_test.dart

```diff
@@ -92,7 +90,7 @@ void main() {
     final VoidCallback listener1 = () { log.add('listener1'); };
     final VoidCallback badListener = () {
       log.add('badListener');
-      throw null;
+      throw null!;
```

Comment:
Won't this throw a null safety exception instead of null, since it's enforcing that null isn't null?

Reply:
you can't actually throw null. This used to throw a NullError, which is just as bogus. But yeah, we should make this throw something better.